src/Elastic.Transport/Components/Providers/RecyclableMemoryStream.cs

```diff
@@ -1,7 +1,3 @@
-// Licensed to Elasticsearch B.V under one or more agreements.
```

Comment:
What is the reason for removing the header from this and the other files?

Reply:
These contain no modifications from us, happy to add these back if needed!

Reply:
Added these back in for all files.